fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -63,6 +63,14 @@ struct ProxyStats {
 	Counter conflictRanges;
 	Counter keyServerLocationIn, keyServerLocationOut, keyServerLocationErrors;
 	Version lastCommitVersionAssigned;
+	double transactionRateAllowed, batchTransactionRateAllowed;
+	double transactionLimit, batchTransactionLimit;
+	// how much of the GRV requests queue was processed in one attempt to hand out read version.
+	double percentageOfGRVQueueProcessed;
+	double percentageOfBatchGRVQueueProcessed;
+
+	LatencySample normalTxnGRVTimeInQueue;
```

Comment:
I think the convention is to refer to the unset priority as "default" priority. I notice that the computations below are including default and system in the normal category, but probably it's worth reporting default priority by itself and optionally reporting system priority. It possibly least interesting to see what system priority is doing, but I'll let you decide whether it's worth it or not.

Reply:
I was trying to follow the convention that combines default and system priority transactions when reporting, but figured I want to call it something other than `TxnGRVTimeInQueue` since that might be suggesting it includes all priorities. But now that I am looking at it again, I guess I'd rather just include default priority transactions and explicitly call it out when logging, just to avoid further confusion.

Reply:
Done.